src/main/java/com/mparticle/kits/AppseeKit.java

```diff
@@ -0,0 +1,219 @@
+        messageList.add(ReportingMessage.fromEvent(this, mpEvent));
+        return messageList;
+    }
+
+    @Override
+    public List<ReportingMessage> logScreen(String s, Map<String, String> map) {
+        Appsee.startScreen(s);
+
+        List<ReportingMessage> messageList = new LinkedList<>();
+        messageList.add(
+                new ReportingMessage(this, ReportingMessage.MessageType.SCREEN_VIEW, System.currentTimeMillis(), null)
+                        .setScreenName(s));
+        return messageList;
+    }
+
+    @Override
+    public void onIdentifyCompleted(MParticleUser mParticleUser, FilteredIdentityApiRequest filteredIdentityApiRequest) {
+
+    }
+
+    @Override
+    public void onLoginCompleted(MParticleUser mParticleUser, FilteredIdentityApiRequest filteredIdentityApiRequest) {
+
+    }
+
+    @Override
+    public void onLogoutCompleted(MParticleUser mParticleUser, FilteredIdentityApiRequest filteredIdentityApiRequest) {
+
+    }
+
+    @Override
+    public void onModifyCompleted(MParticleUser mParticleUser, FilteredIdentityApiRequest filteredIdentityApiRequest) {
+
+    }
+
+    @Override
+    public void onUserIdentified(MParticleUser mParticleUser) {
+        Map<MParticle.IdentityType, String> userIdentities = mParticleUser.getUserIdentities();
+        for (MParticle.IdentityType identityType : userIdentities.keySet()) {
+            if (identityType == MParticle.IdentityType.CustomerId) {
+                Appsee.setUserId(userIdentities.get(identityType));
+                break;
+            }
+        }
+    }
+
+    @Override
+    public void onIncrementUserAttribute(String s, String s1, FilteredMParticleUser filteredMParticleUser) {
+    }
+
+    @Override
+    public void onRemoveUserAttribute(String s, FilteredMParticleUser filteredMParticleUser) {
+    }
+
+    @Override
+    public void onSetUserAttribute(String s, Object o, FilteredMParticleUser filteredMParticleUser) {
+    }
+
+    @Override
+    public void onSetUserTag(String s, FilteredMParticleUser filteredMParticleUser) {
+    }
+
+    @Override
+    public void onSetUserAttributeList(String s, List<String> list, FilteredMParticleUser filteredMParticleUser) {
+    }
+
+    @Override
+    public void onSetAllUserAttributes(Map<String, String> map, Map<String, List<String>> map1, FilteredMParticleUser filteredMParticleUser) {
+    }
+
+    @Override
+    public boolean supportsAttributeLists() {
+        return false;
+    }
+
+    @Override
+    public void onConsentStateUpdated(ConsentState consentState, ConsentState consentState1, FilteredMParticleUser filteredMParticleUser) {
+
+    }
+
+    @TargetApi(Build.VERSION_CODES.O)
+    @Override
+    public void setLocation(Location location) {
+
+        // Since getVerticalAccuracyMeters is supported only for API Level 26+, for previous versions we set the
+        // vertical accuracy to be 0
+        Appsee.setLocation(location.getLatitude(), location.getLongitude(), location.getAccuracy(),
```

Comment:
Can you avoid using the ternary operator? I'd rather it be a little more verbose but easier to read/less error prone.